docs/instant.md

```diff
@@ -637,16 +637,18 @@ This method overrides `Object.prototype.toLocaleString()` to provide a human-rea
 
 The `locales` and `options` arguments are the same as in the constructor to [`Intl.DateTimeFormat`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/DateTimeFormat).
 
+Since `Temporal.Instant` does not carry a time zone, the time zone used for the output will be the `timeZone` property of `options`, if present; and otherwise, the current time zone from the environment.
+
+This is identical to how the time zone is determined in legacy `Date`'s `toLocaleString` method.
+
 Example usage:
 
 ```js
 instant = Temporal.Instant.from('2019-11-18T11:00:00.000Z');
 instant.toLocaleString(); // => example output: 2019-11-18, 3:00:00 a.m.
 instant.toLocaleString('de-DE'); // => example output: 18.11.2019, 03:00:00
-instant.toLocaleString('de-DE', {
-  timeZone: 'Europe/Berlin',
-  weekday: 'long'
-}); // => Montag, 18.11.2019, 12:00:00
+instant.toLocaleString('de-DE', { timeZone: 'Europe/Berlin' });
```

Comment:
It would be helpful for an example to show output with `timeZoneName` enabled.

Reply:
Good call, because that was broken, so I fixed the bug as well.